CMakeLists.txt

```diff
@@ -199,6 +199,10 @@ if(CORE_SYSTEM_NAME STREQUAL osx)
   core_require_dep(Sdl)
 endif()
 
+if(CMAKE_HOST_WIN32 AND NOT CMAKE_SYSTEM_NAME STREQUAL WindowsStore)
+  core_require_dep(detours)
```

Comment:
This is the wrong place.

Reply:
Why? The lines above this is for osx specific checks so this looks like the correct place? Where is the correct one?

Reply:
https://github.com/xbmc/xbmc/blob/34d0c4a94ed60174550d1fefbe82c415326f2ca4/cmake/platform/windows/windows.cmake#L1